be/src/exec/vectorized/hdfs_scan_node.cpp

```diff
@@ -58,14 +58,9 @@ HdfsScanNode::HdfsScanNode(ObjectPool* pool, const TPlanNode& tnode, const Descr
         : ScanNode(pool, tnode, descs) {}
 
 Status HdfsScanNode::_init_table() {
-    if (dynamic_cast<const HdfsTableDescriptor*>(_tuple_desc->table_desc())) {
-        _hdfs_table = dynamic_cast<const HdfsTableDescriptor*>(_tuple_desc->table_desc());
-    } else if (dynamic_cast<const IcebergTableDescriptor*>(_tuple_desc->table_desc())) {
-        _iceberg_table = dynamic_cast<const IcebergTableDescriptor*>(_tuple_desc->table_desc());
-    } else if (dynamic_cast<const HudiTableDescriptor*>(_tuple_desc->table_desc())) {
-        _hudi_table = dynamic_cast<const HudiTableDescriptor*>(_tuple_desc->table_desc());
-    } else {
-        return Status::RuntimeError("invalid table type");
+    _hdfs_table = dynamic_cast<const HdfsBaseTableDescriptor*>(_tuple_desc->table_desc());
```

Comment:
since this scan node can also scan s3/oss. how about name it as dla_table?

Reply:
looks good.